### PR TITLE
Add wildfly weblog variant

### PR DESCRIPTION
**Pull request by DDJavierSantos**
## Description

add wildfly system tests

## Check list

Your PR is not ready to be reviewed? Please save it as a draft :pray:

- [x] Follows the style guidelines of this project (See [how to easily lint the code](https://github.com/DataDog/system-tests/blob/main/docs/edit/lint.md))
- [x] CI is passing
- [x] There is at least one approval from code owners

Yes to all? Feel free to merge it whenever you want :heart:

### CODE REVIEW

**Review by cbeauchesne — APPROVED** [review]

**Comment by cbeauchesne** [comment]
Don't mind about ruby failures. You can merge it when you want !

File: tests/appsec/iast/test_iast_sql_injection.py [hunk 1]
```diff
@@ -2,11 +2,26 @@
...
+from utils import BaseTestCase, interfaces, context, missing_feature, coverage, released, bug
 
 # Weblog are ok for nodejs/express4 and java/spring-boot
 @coverage.basic
-@released(dotnet="?", golang="?", java="?", nodejs="?", php_appsec="?", python="?", ruby="?", cpp="?")
+@released(
+    dotnet="?",
+    golang="?",
+    java={
+        "spring-boot": "0.108.0",
```
Comment (smola) [inline tests/appsec/iast/test_iast_sql_injection.py:13]: These versions for SQL Injection are wrong. They should be 1.1.0. Anyway, I changed this here: https://github.com/DataDog/system-tests/pull/681/files

File: utils/build/docker/java/spring-boot/src/main/java/com/datadoghq/system_tests/springboot/App.java [hunk 2]
```diff
@@ -287,29 +287,15 @@ String raspSSRF(@RequestParam(required = false, name="url") String url) {
     @EventListener(ApplicationReadyEvent.class)
     @Trace
     public void init() {
+        new AppReadyHandler(this).start();
+        /*
         cassandra = new CassandraConnector();
```
Comment (smola) [inline utils/build/docker/java/spring-boot/src/main/java/com/datadoghq/system_tests/springboot/App.java:292]: If we don't need this code, let's remove it, not comment it out.

Comment (DDJavierSantos) [reply utils/build/docker/java/spring-boot/src/main/java/com/datadoghq/system_tests/springboot/App.java]: Done

**Pull request merged**